### ISSUE TITLE
switch_case_on_newline triggered incorrectly on selectors

### ISSUE THREAD

**Issue by hashier** [realm/SwiftLint#1263]
In the given example below `switch_case_on_newline` is incorrectly triggered 

```
override func tableView(_ tableView: UITableView, performAction action: Selector, forRowAt indexPath: IndexPath, withSender sender: Any?) {
    switch action {
    case #selector(copy(_:)):
        <code>
    default:
       break
    }
}
```

**Comment by marcelofabri** [comment]
I couldn't reproduce it with 0.16.1. This was indeed a bug on a previous version (#1020), but the fix (#1023) landed on 0.15. 

Can you confirm you're using the latest version and that this snippet does trigger? Thanks!

**Comment by hashier** [comment]
```
brew info swiftlint
swiftlint: stable 0.16.1 (bottled), HEAD
Tool to enforce Swift style and conventions
https://github.com/realm/SwiftLint
/usr/local/Cellar/swiftlint/HEAD-1b82909 (34 files, 9.9M) *
  Built from source on 2016-11-12 at 18:08:41
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/swiftlint.rb
==> Requirements
Build: git ✔
Required: xcode ✔
```

![screen shot 2017-01-30 at 14 37 58](https://cloud.githubusercontent.com/assets/92332/22425084/ba75e0d0-e6f9-11e6-964c-3874512c78ab.png)

**Comment by hashier** [comment]
Is there a way to ask swiftlint directly for the version?
`case invalidResponse([String: AnyObject])` seems to broken as well... I will try to use the swiftlint from pods

**Comment by marcelofabri** [comment]
> Is there a way to ask swiftlint directly for the version?

I think `swiftlint version` is what you want.

You can try to run from the terminal as well to be sure that you're using the right binary.

**Comment by hashier** [comment]
Using the pods version works.
`brew uninstall -f swiftlint` `brew install swiftlint` works now as well...

**Comment by marcelofabri** [comment]
Thanks for the feedback, @hashier!
